Dockerfile

```diff
@@ -432,8 +432,8 @@ FROM scratch AS ortbin
 COPY --from=ortbuild /opt/ort /opt/ort
 
 #------------------------------------------------------------------------
-# Main Minimal Runtime container
-FROM base as run
+# Minimal Runtime container
+FROM base as minimal
```

Comment:
Because all builds relies on minimal target.  If you not do that, will break the build of all components. Is all about context usage.
Will not break the build, will just make it huge slow, as every sngle component will buils docker from scratch, like no cached exists